mlbench_core/utils/pytorch/helpers.py

```diff
@@ -177,7 +172,7 @@ def config_pytorch(use_cuda=False, seed=None, cudnn_deterministic=False):
     return rank, world_size, graph
 
 
-def log_metrics(run_id, rank, epoch, metric_name, value):
+def log_metrics(run_id, rank, epoch, metric_name, value, tracker=None, time=None):
```

Comment:
log_metrics is deprecated, change https://github.com/mlbench/mlbench-core/blob/develop/mlbench_core/utils/pytorch/helpers.py#L182 instead.
log_metrics has performance implications since it can take quite some time (up to 5% of all running time) to create the ApiClient, `LogMetrics` uses a cached ApiClient (this only uses 0.1% of total runtime in my experiments).